recipe/meta.yaml

```diff
@@ -81,7 +81,7 @@ requirements:
 test:
   requires:
     - pip
-    - hypothesis >=5.5.3
+    - hypothesis >=6.13.0
     - pytest >=6.0
     - pytest-xdist >=1.31
   imports:
```

Comment:
Could you remove or comment `test/imports` because we already test it in `run_test.py`, see https://github.com/AnacondaRecipes/pandas-feedstock/blob/b37f7d8f8f5e50c6dcd6fbe1650acd8619a1651b/recipe/run_test.py#L1
It will resolve anaconda-linter error:
`ERROR: /tmp/abs_abfys8aluk/clone/recipe/meta.yaml:88: has_imports_and_run_test_py: Imports and python test file cannot coexist.`